util.go

```diff
@@ -354,3 +369,22 @@ func fmtTime(u *User, lastStamp time.Time) string {
 	}
 	return lastStamp.In(u.Timezone.Location).Format("3:04")
 }
+
+// See if the public key is there and if it is not, try to create.
+func checkKey(keyPath string) {
```

Comment:
This function returns silently both when keys are already there and when there is a legitimate error. Those legitimate errors need to be communicated to the user. Also, if we do end up trying to generate keys, let the user know that keys were made and saved in [paths]